nixos/modules/services/network-filesystems/moosefs.nix

```
@@ -168,29 +144,61 @@ in {
             options.DATA_PATH = lib.mkOption {
               type = lib.types.str;
               default = "/var/lib/mfs";
-              description = "Directory for lock file.";
+              description = "Directory for lock files and other runtime data.";
             };
           };
+          description = "Chunkserver configuration options (mfschunkserver.cfg).";
+        };
+      };
+
+      cgiserver = {
+        enable = lib.mkEnableOption ("MooseFS CGI server for web interface");
```

```suggestion
        enable = lib.mkEnableOption "MooseFS CGI server for web interface";
```

There should also be a warning here: anyone with access to the CGI interface can not only see all the statistics, but also put chunk servers into maintainance mode (i.e. take them out-of-service). One should not turn on the CGI interface without further measures to protect it from unauthorized access.

---

Yes but the master is itself a critical server I don't see why this is more dangerous.
The CGI is available if you don't want it by default.

---

I mean: add a note to the text in the enable option, warning that the cgi server should be protected from unauthorized access.